editor/src/components/editor/editor-component.tsx

```diff
@@ -362,9 +362,8 @@ export const EditorComponentInner = React.memo((props: EditorProps) => {
           <LowPriorityStoreProvider>
             {(isChrome as boolean) ? null : <BrowserInfoBar />}
             <LoginStatusBar />
-            {unless(isFeatureEnabled('Draggable Floating Panels'), <TitleBar />)}
+            {/* TODO make sure the title bar and the navigator and the inspector are in the Low Priority Store! */}
```

Comment:
Navigator and Inspector are. What do you mean by "title bar" here?

Reply:
"title bar" must have been a brain fart.
```suggestion
```